server-config.testnet.json

```diff
@@ -1,13 +1,14 @@
 {
-    "url": "https://dev.relay.rifcomputing.net:8090",
-    "port": 8090,
-    "relayHubAddress": "0x66Fa9FEAfB8Db66Fe2160ca7aEAc7FC24e254387",
-    "relayVerifierAddress": "0x56ccdB6D312307Db7A4847c3Ea8Ce2449e9B79e9",
-    "deployVerifierAddress": "0x5C6e96a84271AC19974C3e99d6c4bE4318BfE483",
-    "gasPriceFactor": 1,
-    "rskNodeUrl": "http://172.17.0.1:4444",
-    "devMode": true,
-    "customReplenish": false,
-    "logLevel": 1,
-    "workdir": "/srv/app/environment"
+  "url": "https://dev.relay.rifcomputing.net:8090",
+  "port": 8090,
+  "relayHubAddress": "0x66Fa9FEAfB8Db66Fe2160ca7aEAc7FC24e254387",
+  "relayVerifierAddress": "0x56ccdB6D312307Db7A4847c3Ea8Ce2449e9B79e9",
+  "deployVerifierAddress": "0x5C6e96a84271AC19974C3e99d6c4bE4318BfE483",
+  "gasPriceFactor": 1,
+  "rskNodeUrl": "http://172.17.0.1:4444",
+  "devMode": true,
+  "customReplenish": false,
+  "workerFeePercentage": "0",
```

Comment:
I would call it just `feePercentage`. Not necessary the fee will be collected by the worker.

Reply:
done